### DISCUSSION TITLE
Graph: Hover tooltip sort order to match stacked series

### DISCUSSION THREAD

**Discussion by belm0** [grafana/grafana#38160] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
"Panel -> Display -> hover tooltip -> sort order" should have an option to matched the "stacked" series ordering.

Currently "sort order: None" seems to actually mean "in query order".  Since the "stacked" option causes the series to be in query order from bottom to top, "None" will cause the hover order to be reversed with respect to the stacked series.

Proposal 1:
  * rename "None" to "query order"
  * add "reverse query order" (intended for use with stacked)

Proposal 2:
  * keep the "None" name as-is, but when stacked series are enabled, have it automatically use the reverse query order

**Why is this needed**:
It should be possible to make the ordering in hover tooltip match the ordering of stacked series.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!

**Comment by belm0** [comment]
It was originally a [github issue](https://github.com/grafana/grafana/issues/28206).  

"As you may have heard, every few years Grafana changes our issue tracking platform, such that we can summarily close user reports and feedback and pretend we're reducing the open issue count."  @gelicia
